diff --git a/node_modules/graphql/error/locatedError.js b/node_modules/graphql/error/locatedError.js
index 658c439..5dd6387 100644
--- a/node_modules/graphql/error/locatedError.js
+++ b/node_modules/graphql/error/locatedError.js
@@ -20,7 +20,7 @@ function locatedError(rawOriginalError, nodes, path) {
   var _nodes;
 
   // Sometimes a non-error is thrown, wrap it as an Error instance to ensure a consistent Error interface.
-  var originalError = rawOriginalError instanceof Error ? rawOriginalError : new Error('Unexpected error value: ' + (0, _inspect.default)(rawOriginalError)); // Note: this uses a brand-check to support GraphQL errors originating from other contexts.
+  var originalError = rawOriginalError; // Note: this uses a brand-check to support GraphQL errors originating from other contexts.
 
   if (Array.isArray(originalError.path)) {
     return originalError;
